nimbus-ui/nimbusui/src/app/services/service.constants.ts

```diff
@@ -113,7 +113,9 @@ export class ServiceConstants {
             required: 'Field is required.',
             pattern: 'Field does not match the required pattern.',
             minMaxSelection: 'Field does not meet min/max requirement.',
-            isNumber: 'Value must be a number.'
+            isNumber: 'Value must be a number.',
+            isPast: 'Value Must be past.',
+            isFuture: 'Value Must be Future.'
```

Comment:
Please change to: `isPast: 'Value must be after today's date.'`

Reply:
It can be today also right? can i change the default message to
It must be today or after today's date.